x-pack/legacy/plugins/uptime/public/uptime_app.tsx

```diff
@@ -166,47 +164,21 @@ const Application = (props: UptimeAppProps) => {
                       <UptimeSettingsContext.Provider value={initializeSettingsContextValues()}>
                         <EuiPage className="app-wrapper-panel " data-test-subj="uptimeApp">
                           <main>
-                            <EuiFlexGroup
-                              alignItems="center"
-                              justifyContent="spaceBetween"
-                              gutterSize="s"
-                            >
-                              <EuiFlexItem>
-                                <EuiTitle>
-                                  <h1>{headingText}</h1>
-                                </EuiTitle>
-                              </EuiFlexItem>
-                              <EuiFlexItem grow={false}>
-                                <UptimeDatePicker
-                                  refreshApp={refreshApp}
+                            <Switch>
+                              <Route path="/monitor/:monitorId/:location?">
```

Comment:
Like I mentioned above, if we're using this string multiple places to refer to the route we should probably extract it and reference it from a common location.

Reply:
done !!

Reply:
Yes, this is much cleaner! Thank you 👍